src/Symfony/Bundle/SecurityBundle/composer.json

```diff
@@ -34,7 +34,7 @@
         "symfony/process": "~2.0,>=2.0.5",
         "symfony/validator": "~2.2",
         "symfony/yaml": "~2.0,>=2.0.5",
-        "twig/twig": "~1.20|~2.0"
+        "twig/twig": "~1.23|~2.0"
```

Comment:
Is this really needed? The `LogoutUrlExtension` does not make use of the `setEnvironment()` method.